src/libpisp/frontend/frontend.cpp

```diff
@@ -399,7 +399,9 @@ void FrontEnd::Prepare(pisp_fe_config *config)
 		{
 			pisp_image_format_config &image_config = fe_config_.ch[i].output.format;
 
-			getOutputSize(i, image_config.width, image_config.height);
+			uint16_t width = image_config.width;
+			uint16_t height = image_config.height;
+			getOutputSize(i, width, height);
```

Comment:
We need to copy width/height to image_config here.

However, I think the easiest thing would be to remove this patch and add ``-Wno-address-of-packed-member`` to the compiler flags.